sdk/containerregistry/Microsoft.Azure.ContainerRegistry/src/Customizations/AzureContainerRegistryClientExtended.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.Rest;
+using Microsoft.Rest.Azure;
+using System.Net.Http;
+namespace Microsoft.Azure.ContainerRegistry
+{
+    public partial class AzureContainerRegistryClient : ServiceClient<AzureContainerRegistryClient>, IAzureContainerRegistryClient, IAzureClient
+    {
+
+        // MANUALLY ADDED FOR INTERNAL TEST PURPOSES
```

Comment:
Which part was added for testing? Is that still required?

Reply:
Oh that one is a tad confusing. Essentially the test framework requires one of the constructors there (internal AzureContainerRegistryClient(System.Uri loginUri, ServiceClientCredentials credentials, params DelegatingHandler[] handlers)) in order to run at all but autorest wasnt generating it (pretty sure it never did) so I added it so that the tests can be run. We thus do need to keep it but the comment is misleading. Should probably read "For Testing Framework use only"